### PR TITLE
Fixed it

### PR DESCRIPTION
**Pull request by KathyGray**
Changed line 55 back to 'public'
Sourced all images to /math105/
All ran on my computer.

### CODE REVIEW

File: views/math105/labs/lab01.pug [hunk 1]
```diff
@@ -44,10 +43,10 @@ block content
 
         p.
 
-        a: img(src="/math105/roller_coaster.PNG" alt="Roller Coaster Data")
+        a: img(src="/roller_coaster.PNG" alt="Roller Coaster Data", style='width:600px;height:250px;')
```
Comment (roualdes) [inline views/math105/labs/lab01.pug]: should be src="/math105/roller_coaster.PNG"

**Review by roualdes — CHANGES_REQUESTED** [review]
I think there's just one more src path to change. I tried to mark it, but some of this is new to me too, so it's in `views/math105/labs/lab01.pug`.

**Comment by roualdes** [comment]
For learning's sake: you could have reopened PR #2 and push your latest updates to it.  Either way works, just letting you know.

**Comment by KathyGray** [comment]
OK. Should be good to go.

**Pull request merged**